lib/elixir/lib/calendar/datetime.ex

```diff
@@ -1084,6 +1084,16 @@ defmodule DateTime do
       Calendar.ISO.time_to_string(hour, minute, second, microsecond, format)
   end
 
+  def from_iso8601(string, format_or_calendar)
```

Comment:
Let's move the docs to this function and have some short docs on `from_iso8601`, as I mentioned in the previous comment. WDYT? Thank you.

Reply:
You also need to define the default argument here:

```suggestion
  def from_iso8601(string, format_or_calendar \\ Calendar.ISO)
```

And remove the default args from the arity 3!

Reply:
Thanks!